tests/unit/shared-adapter-behaviour.js

```diff
@@ -1654,15 +1654,21 @@ SharedBehavior.mockUpdateWithErrorMessages = function () {
 
       mockUpdate(profile).fails({
         status: 400,
-        response: { errors: { description: ['invalid data'] } },
+        /* ember-data 3.28 -> 4.x rest adapter normalizeErrorResponse only passes on string response value as 'detail'
+         * ember-data 5.0 fixes this, but we cannot use it yet https://github.com/emberjs/data/pull/8621
+         * Just use a primitive value in the meantime.
+         */
+        // response: { errors: { description: ['invalid data'] } },
+        response: 'invalid data',
```

Comment:
ember-data-model-fragments is holding us back from ember-data 5 which contains the fix we want. This is a pragmatic solution to keep the bulk of the test and move forward